### PR TITLE
[ADD] web_widget_digitized_signature: backport from 10.0

### PR DESCRIPTION
**Pull request by cubells**
- Merge web_widget_digitized_signature and web_widget_digitized_signature_user modules
- Update views
- Update README

cc @Tecnativa

### CODE REVIEW

**Comment by yajo** [comment]
OK, please advise when WIP finishes to label properly 😉

**Comment by yajo** [comment]
Is this still WIP? Travis is ❌ BTW

**Comment by pedrobaeza** [comment]
Not in theory, but a good squashing is needed

**Comment by cubells** [comment]
It's not WIP.

File: web_widget_digitized_signature/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# Copyright 2011-2015 Serpent Consulting Services Pvt. Ltd.
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Web Widget Digitized Signature",
+    "version": "9.0.1.0.0",
+    "author": "Serpent Consulting Services Pvt. Ltd., "
+               "Agile Business Group, "
+               "Odoo Community Association (OCA)",
```
Comment (rafaelbn) [inline web_widget_digitized_signature/__openerp__.py]: missing Tecnativa

**Review by rafaelbn — APPROVED** [review]
- Tested functionally 👍 
- Why travis fails?
- Add author Tecnativa

File: web_widget_digitized_signature/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+# Copyright 2004-2010 OpenERP SA (<http://www.openerp.com>)
+# Copyright 2011-2015 Serpent Consulting Services Pvt. Ltd.
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Web Widget Digitized Signature",
+    "version": "9.0.1.0.0",
+    "author": "Serpent Consulting Services Pvt. Ltd., "
+               "Agile Business Group, "
```
Comment (chienandalu) [inline web_widget_digitized_signature/__openerp__.py]: Add Tecnativa

**Review by chienandalu — APPROVED** [review]
- Tested on runbot :+1: 
- Travis error seems to come from another module test (web_widget_float_formula).

**Review by yajo — APPROVED** [review]
Code & functional review, it's almost perfect. Just some points:

- Travis is ❌ 
- All fields from `res.partner` are inherited in `res.users`, so do you think it would make sense to add the field in `res.partner`, just in case we want to add it for non-users for whatever reason? Maybe it makes no sense and it's OK as it is, just asking...
- Do you think you could improve coverage? [It seems you don't need much](https://codecov.io/gh/OCA/web/compare/9f77ba21250788ad6c3087109b2adac63a3e3eea...90647ccb1c6d3c8c1f733ec72899642863285cc3/diff#D11-22) to get to 100%...

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**